cips/cip-PFM.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Packet Forward Middleware
+description: Adopt Packet Forward Middleware for multi-hop IBC and path unwinding
+author: Alex Cheng (@akc2267)
+discussions-to: URL
+status: Draft
+type: Standards Track
+category: Core
+created: Date created on, in ISO 8601 (yyyy-mm-dd) format
+---
+<!-- markdownlint-disable MD013 -->
+
+## Abstract
+
+This CIP integrates Packet Forward Middleware, the IBC middleware that enables multi-hop IBC and path unwinding to preserve fungibility for IBC-transferred tokens.
+
+## Specification
+
+_The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174._
+
+- Celestia MUST import and integrate Packet Forward Middleware. 
```

Comment:
Can we have a copy-pasta definition of Packet Forward Middleware and what it does as part of the specification? You can use the terminology linked here: https://github.com/cosmos/ibc-apps/tree/modules/async-icq/v7.1.1/middleware/packet-forward-middleware